ui/components/multichain/pages/send/components/hex.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useI18nContext } from '../../../../../hooks/useI18nContext';
+import { Label } from '../../../../component-library';
+import { getSendHexData, updateSendHexData } from '../../../../../ducks/send';
+import { SendPageRow } from '.';
+
+export const SendHexData = () => {
+  const t = useI18nContext();
+  const dispatch = useDispatch();
+
+  const data = useSelector(getSendHexData);
+
+  return (
+    <SendPageRow>
+      <Label>{t('hexData')}</Label>
+      <textarea
+        onInput={(event: Event & { target: HTMLTextAreaElement }) => {
+          const newData = event.target.value.replace(/\n/gu, '') || null;
+          dispatch(updateSendHexData(newData ?? ''));
+        }}
+        placeholder={t('optional')}
+        defaultValue={data || ''}
+      />
```

Comment:
Can we replace textarea with the `TextField` component from DS ?

Reply:
`TextField` doesn't work here, as it only abstracts `Input`